### DISCUSSION TITLE
CoPilot suddenly not working

### DISCUSSION THREAD

**Discussion by Amanston** [community/community#7623] [Copilot Conversations]
I've been using CoPilot for the last 2 weeks for C# and Unity and for some reason today after going away for a break and coming back to my computer copilot is no longer offering any kind of suggestions

I've even tried typing out comments to make a simple Debug.log and it couldn't seem to synthesize any kind of solution.

I'm a bit fo a novice so I'm not seeing if there was anyt clear reason that copilot may be having any kind of issues.

Any info on thing I can try to get it working again would be greatly appreciated

**Comment by davecheney** [comment]
What do you see in the GitHub Copilot output window?

**Reply by Amanston** [reply]
> 
> 
> What do you see in the GitHub Copilot output window?

Hey Dave,

Seems like it's started working again for me. 

Could you tell me how to check the copilot output? I'm loving this so far and want to be able to give more feeback

cheers!

**Reply by davecheney** [reply]
View -> output; choose GitHub copilot from the pull down.
